src/loki_alert_rules/openfga_high_severity_log.rule

```diff
@@ -0,0 +1,9 @@
+groups:
+- name: OpenFGAHighSeverityLog
+  rules:
+  - alert: HighFrequencyHighSeverityLog
+    expr: sum by(level) (count_over_time({%%juju_topology%%} | json | __error__ != "JSONParserErr" | level =~ `error|fatal|critical` [5m])) > 100
```

Comment:
did u manage to understand how to parse the logs?

Reply:
Yes, Loki LogQL doc helped
Also, the way pebble output openfga logs confused me at first